libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -241,6 +241,12 @@ bool AC_Fence::check_fence_circle()
         return false;
     }
 
+    Vector2f home;
+    if (_ahrs.get_relative_position_NE_home(home)) {
+        // we (may) remained breached if we can't update home
```

Comment:
remained -> remain?

*We may remained breached* sounds strange to me. But *We may remain breached* and *We remain breached* looks good,